src/main/java/eu/dissco/backend/domain/annotation/batch/BatchMetadata.java

```diff
@@ -7,7 +7,7 @@
 
 @Value
 public class BatchMetadata {
-  static Integer placeInBatch = 1;
+  Integer placeInBatch = 1;
```

Comment:
What does this change exactly?

Reply:
removing `static` means that it searlizes into a json, otherwise the term is ignored when batchMetadata is serialized. We send batchMetadata as a json to the specimen processor, and it's an important part of the pipeline.